modules/openapi-generator/src/main/resources/elixir/README.md.mustache

```diff
@@ -2,33 +2,39 @@
 
 {{appDescription}}
 
-### Building
+## Building
 
 To install the required dependencies and to build the elixir project, run:
-```
+
+```console
 mix local.hex --force
 mix do deps.get, compile
 ```
 
 ## Installation
 
-If [available in Hex](https://hex.pm/docs/publish), the package can be installed
-by adding `{{#underscored}}{{packageName}}{{/underscored}}` to your list of dependencies in `mix.exs`:
+If [available in Hex][], the package can be installed by adding `{{#underscored}}{{packageName}}{{/underscored}}` to
+your list of dependencies in `mix.exs`:
 
 ```elixir
-def deps do
-  [{:{{#underscored}}{{packageName}}{{/underscored}}, "~> 0.1.0"}]
-end
+{{=<% %>=}}def deps do
+  [{<%#atom%><%#underscored%><%packageName%><%/underscored%><%/atom%>, "~> <%appVersion%>"}]
+end<%={{ }}=%>
 ```
 
-Documentation can be generated with [ExDoc](https://github.com/elixir-lang/ex_doc)
-and published on [HexDocs](https://hexdocs.pm). Once published, the docs can
-be found at [https://hexdocs.pm/{{#underscored}}{{packageName}}{{/underscored}}](https://hexdocs.pm/{{#underscored}}{{packageName}}{{/underscored}}).
-
+Documentation can be generated with [ExDoc][] and published on [HexDocs][]. Once published, the docs can be found at
+[https://hexdocs.pm/{{#underscored}}{{packageName}}{{/underscored}}][docs].
 
 ## Configuration
 
-You can override the URL of your server (e.g. if you have a separate development and production server in your configuration files.
+You can override the URL of your server (e.g. if you have a separate development and production server in your
+configuration files).
+
 ```elixir
-config :{{#underscored}}{{appName}}{{/underscored}}, base_url: "{{{basePath}}}"
-```
\ No newline at end of file
+config :{{#underscored}}{{packageName}}{{/underscored}}, base_url: "{{{basePath}}}"
+```
+
```

Comment:
Would it make sense to also add something like this:

Or, in case you would like to have some different clients with different URLs, you can provide the base_url during the new call
```elixir
client = {{moduleName}}.Connection.new(base_url: ""{{{basePath}}}")
```

Reply:
Yes, I think that would work as well. I’ll look at updating with your additional changes.